src/core/friendly_errors/sketch_reader.js

```diff
@@ -0,0 +1,396 @@
+        //is OK so we will skip the current iteration and check for the
+        //next element.
+        continue;
+      }
+      //if we are not getting an error this means
+      //user have changed the value. We will check
+      //if the value is changed and if it is changed
+      //then report.
+      if (constants[tempArray[i]] !== element) {
+        let url = `https://p5js.org/reference/#/p5/${tempArray[i]}`;
+        p5._friendlyError(
+          translator('fes.sketchReaderErrors.reservedConst', {
+            url: url,
+            symbol: tempArray[i]
+          })
+        );
+        isFound = true;
+        break;
+      }
+    }
+    //if a p5.js constant is already reported then no need to check
+    //for p5.js functions.
+    if (!isFound) {
+      //the below code gets a list of p5.js functions
+      let p5Constructors = {};
+      for (let key of Object.keys(p5)) {
+        // Get a list of all constructors in p5. They are functions whose names
+        // start with a capital letter
+        if (typeof p5[key] === 'function' && key[0] !== key[0].toLowerCase()) {
+          p5Constructors[key] = p5[key];
+        }
+      }
+      const keyArray = Object.keys(p5Constructors);
+      let functionArray = [];
+      //get the names of all p5.js functions
+      for (let i = 0; i < keyArray.length; i++) {
+        functionArray.push(
+          ...Object.keys(p5Constructors[keyArray[i]].prototype)
+        );
+      }
+      functionArray = functionArray.filter(ele => !ele.includes('_'));
+
+      //we have p5.js function names with us so we will check
+      //if they have been declared or not.
+      for (let i = 0; i < functionArray.length; i++) {
+        //ignoreFunction contains the list of functions to be ignored
+        if (!ignoreFunction.includes(functionArray[i])) {
+          try {
+            //if we get an error that means the function is not declared
+            element = eval(functionArray[i]);
+          } catch (e) {
+            //we will skip the iteration
+            continue;
+          }
+          //if we are not getting an error this means
+          //user have used p5.js function. Check if it is
+          //changed and if so then report it.
+          let k = 0;
+          for (; k < keyArray.length; k++) {
+            if (
+              p5Constructors[keyArray[k]].prototype[functionArray[i]] ===
+              undefined
+            );
+            else {
+              if (
+                p5Constructors[keyArray[k]].prototype[functionArray[i]] !==
+                element
+              ) {
+                p5._friendlyError(
+                  translator('fes.sketchReaderErrors.reservedFunc', {
+                    symbol: functionArray[i]
+                  })
+                );
+                isFound = true;
+                break;
+              }
+            }
+          }
+          if (k < keyArray.length) break;
```

Comment:
I think this line is a bit confusing - using `isFound` to break the loop is clearer (if that's possible). In fact, you could just `return true` straight from line 344

Reply:
Great suggestion I will make the changes

Reply:
This is a great suggestion thanks, I will change the code and push it soon